src/cbor_builders.cc

```diff
@@ -65,9 +65,9 @@ void MakeCredentialCborBuilder::SetDefaultClientDataHash() {
 }
 
 void MakeCredentialCborBuilder::SetDefaultPublicKeyCredentialRpEntity(
-    const std::string& rp_id) {
+    std::string&& rp_id) {
```

Comment:
I had pass-by-value in mind, which if I'm not mistaken is strictly better than taking r-value reference, since it's possible to call with `foo(std::move(x))` if ownership is passed or `foo(x)` to create a local copy.

https://abseil.io/tips/117

Reply:
Changed, but curious: If I'm just writing `foo(std::string s)` and calling `foo(temp())`, why is that value not copied? I thought this is why you want `foo(std::string&& s)`, so you don't need a copy there.

Reply:
My understanding is that it's copy-elision as described in the abseil tip, so the result of `temp()` is actually the object `foo` has as argument.

Reply:
Okay, sounds good then.